app/actions/ConfigActionCreators.js

```diff
@@ -0,0 +1,25 @@
+import constants from '../constants';
+import ConfigAPI from '../api/ConfigAPI';
+
+export default class ConfigActionCreators {
```

Comment:
You can see an example on the app I'm building here: https://github.com/benmvp/bart-salmon/blob/master/app/actions/index.js

The only difference is that I am using async functions